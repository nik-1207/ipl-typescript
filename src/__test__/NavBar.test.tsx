import React from "react";
import renderer from "react-test-renderer";
import Navbar from "../components/NavBar";

it("renders correctly", () => {
  const snapshot = renderer.create(<Navbar />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
