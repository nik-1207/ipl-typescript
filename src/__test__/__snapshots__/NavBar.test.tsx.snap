// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar-0-2-2"
  title="navbar"
>
  <img
    alt="ipl-logo"
    className="logoImage-0-2-1"
    src="ipl-logo.png"
  />
  <h1
    className="navHeading-0-2-3"
    title="heading"
  >
    INDIAN PREMIRE LEAGUE
  </h1>
</nav>
`;
